pkg/audit/service.go

```diff
@@ -73,31 +75,33 @@ func (service *Service) Run(ctx context.Context, interval time.Duration) (err er
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	errch := make(chan error)
```

Comment:
Make this channel buffered otherwise it can leak the goroutine. Also is the goroutine here even necessary?

Reply:
```
for {
    select {
    case <-ticker.C:
        stripe, err := service.Cursor.NextStripe(ctx)
        if err != nil {
            return err
        }

        authorization, err := service.Cursor.pointers.SignedMessage()
        if err != nil {
            return err
        }
        
        ...

    case <-ctx.Done():
        return ctx.Err()
    }
}

return nil
```

Reply:
I'm not sure the reason for using a goroutine, I just know that the overlay's service using a ticker also uses a goroutine. But I can go ahead and take it out?